translations/strings.yaml

```diff
@@ -790,6 +790,9 @@ global:
   paste:
     value: Paste
     comment: Label for button to paste value from clipboard into input
+  clearAll:
```

Comment:
This is not a string you want to reuse in different contexts, "all" needs to be adapted to the noun it refers to, which might have different genders.

This should be moved out of global and into its specific section for app exclusions. The ID should be more specific, e.g. `clearAllApps`, and the comment updated accordingly.